transform/gtex/samples.py

```diff
@@ -0,0 +1,54 @@
+from bmeg.emitter import JSONEmitter
+from bmeg.ioutils import read_tsv
+from bmeg.vertex import Aliquot, Biosample, Project, Individual
+from bmeg.edge import AliquotFor, BiosampleFor, InProject
+
+
+def extract_individual_id(sample_id):
+    return "-".join(sample_id.split("-")[:2])
+
+
+emitter = JSONEmitter("gtex")
+samples = read_tsv("source/gtex/GTEx_v7_Annotations_SampleAttributesDS.txt")
```

Comment:
Not sure how I feel about hard coding the input paths in the code. At the very least the original download URLs should be documented somewhere. I'm assuming this file is coming from https://storage.googleapis.com/gtex_analysis_v7/annotations/GTEx_v7_Annotations_SubjectPhenotypesDS.txt

Reply:
This is the pattern we are considering adopting for all transformers. Check out the style guide PR for more details https://github.com/bmeg/bmeg-etl/pull/160.